library/src/scala/runtime/stdLibPatches/Predef.scala

```diff
@@ -72,8 +72,9 @@ object Predef:
    *    TC { type Self = A }
    *
    *  which is what is needed for a context bound `[A: TC]`.
+   *
+   *  @experimental added by compiler in Definitions.patchStdLibClass
    */
-  @experimental
   infix type is[A <: AnyKind, B <: Any{type Self <: AnyKind}] = B { type Self = A }
```

Comment:
does it still appear experimental in TASTy? otherwise I guess we'd have to make this PR `needs-minor-release` to avoid leaking to prior compilers in same minor

Reply:
It will hopefully get into 3.5

Reply:
> It will hopefully get into 3.5

"self" and "is" both will be added in scala 3.5?